.gitignore

```diff
@@ -2,3 +2,4 @@
 /build/
 /composer.lock
 /vendor/
+.idea/
```

Comment:
This is ide-specific, and should be part of your own global `.gitignore` file, rather than a project setting